blogposts/2021/workspaces-of-sourcegraph.md

```diff
@@ -0,0 +1,187 @@
+- Dark mode (except for Slack and Chromium).
+- [Atom One Dark](https://marketplace.visualstudio.com/items?itemName=akamud.vscode-theme-onedark) themes for VSCode, IntelliJ, i3/Rofi and Kitty terminal and [modified Adapta Nokto](https://github.com/Strum355/adapta-gtk-theme) for GTK applications.
+- Dual monitor with 27" 1440p and 24" 1080p monitors.
+- Custom spin of Inconsolata (named [MonoStroom](https://github.com/Strum355/MonoStroom)) as system-wide monospace font.
+- Big time user of LSP features (find symbol, find refs/impls etc) and keyboard shortcuts where possible (with vimium for chromium).
+
+![Workspace 1 of Noah Santschi-Cooney, Software Engineer](https://storage.googleapis.com/sourcegraph-assets/blog/workspaces-of-sourcegraph/noah-sc-1.png)
+![Workspace 2 of Noah Santschi-Cooney, Software Engineer](https://storage.googleapis.com/sourcegraph-assets/blog/workspaces-of-sourcegraph/noah-sc-2.png)
+
+## Eric Fritz, Software Engineer
+
+- VSCode with [Fira Code](https://github.com/tonsky/FiraCode) and [Nord Extra Dark](https://marketplace.visualstudio.com/items?itemName=yamenarahman.nord-extra-dark), [Rainbow Brackets](https://marketplace.visualstudio.com/items?itemName=2gua.rainbow-brackets), and [Indent-Rainbow](https://marketplace.visualstudio.com/items?itemName=oderwat.indent-rainbow)
+- Dark mode
+- 27" 5k iMac (3.2 GHz 8-core Intel Xeon W, 64GB RAM) and 2x Dell Ultrasharp U2718Q 27" 4K IPS monitors Portrait/Landscape/Portrait mode
+- Ergodox EZ with Cherry MX Blue switches
+- My [dotfiles](https://github.com/efritz/dotfiles)
+
+![Combined workspace of Eric Fritz, Software Engineer](https://storage.googleapis.com/sourcegraph-assets/blog/workspaces-of-sourcegraph/efritz-combined.png)
+
+## Adam Harvey, Software Engineer
+
+I use a mixture of VS Code with more or less the recommended plugins and nothing else (sometimes in remote mode to a devbox in the cloud; sometimes locally), and neovim with airline (for a better statusbar) and coc (for LSP purposes) — I've tried the native support in neovim nightlies, but I hit too many rough edges. Everything is in light mode. Like Keegan, I'm also very keyboard driven.
+
+I don't tend to layer much else into my editing experience: I use a separate terminal (kitty), rather than the builtin ones.
+
+Outside of that, I tend to be very single window focused: I run everything in a separate fullscreen window on a separate workspace in Sway, and use hotkeys to switch between workspaces. (1 is always Firefox; 2 is kitty; 3 is personal chat; 4 is Slack; 5 is VSCode; 6- are used for ad hoc one offs.) Generally speaking I have as little chrome as possible: menu bars are usually disabled once I know hot keys, and I have sway configured to disable titlebars unless the workspace is split or the window is floating. Minimal distraction.
+
+![Workspace 1 of Adam Harvey, Software Engineer](https://storage.googleapis.com/sourcegraph-assets/blog/workspaces-of-sourcegraph/aharvey.png)
```

Comment:
Cleaning up wording, capitalisation, and linking things (debating whether to put a "btw i use arch" joke in):

```suggestion
I use a mixture of [Visual Studio Code](https://code.visualstudio.com/) with more or less the recommended language support plugins and nothing else (sometimes in remote mode to a devbox in the cloud; sometimes locally), and [Neovim](https://neovim.io/) with [airline](https://github.com/vim-airline/vim-airline) for a better statusbar and [coc.nvim](https://github.com/neoclide/coc.nvim) to get the same LSP magic as you get in VSCode — I've tried the [native LSP client](https://neovim.io/doc/user/lsp.html) in neovim nightlies, but I hit too many rough edges. I tend to keep my editors as just editors: I don't use builtin terminals or debugging capabilities.

Styling wise, everything is in light mode — I find it helps me focus, and I keep dark mode for late night phone usage when I should be sleeping — and I use [Fira Code](https://github.com/tonsky/FiraCode) as a font because I love ligatures, even if I don't always love its serifs. Like Keegan, I'm also very keyboard driven.

In terms of general workflow, I tend to be very single window focused: I run everything in a separate fullscreen window on a separate workspace in [Sway](https://swaywm.org/) on [Arch Linux](https://archlinux.org/), and use hotkeys to switch between workspaces. (1 is always Firefox; 2 is kitty; 3 is personal chat; 4 is Slack; 5 is VSCode; 6- are used for ad hoc one offs.) Generally speaking I have as little chrome as possible, besides [Waybar](https://github.com/Alexays/Waybar): menu bars are usually disabled once I know hot keys, and I have Sway configured to disable title bars unless the workspace has multiple tiled windows or the window is floating. Minimal distraction.

![vim workspace of Adam Harvey, Software Engineer](https://storage.googleapis.com/sourcegraph-assets/blog/workspaces-of-sourcegraph/aharvey-1.png)
![VSCode workspace of Adam Harvey, Software Engineer](https://storage.googleapis.com/sourcegraph-assets/blog/workspaces-of-sourcegraph/aharvey-2.png)
```

I'm also going to put a better screenshot in.